src/core/Akka/Actor/ActorSystem.cs

```diff
@@ -278,6 +278,10 @@ public static ActorSystem Create(string name)
 
         private static ActorSystem CreateAndStartSystem(string name, Config withFallback, ActorSystemSetup setup)
         {
+            // allows the ThreadPool to scale up / down dynamically
+            // by removing minimum thread count, which in our benchmarks
+            // appears to negatively impact performance
+            ThreadPool.SetMinThreads(0, 0); 
```

Comment:
The differences in the benchmarks with this and without it are massive. However, if this causes problems for users they can easily reset it by calling `ThreadPool.SetMinThreads(yourValue, yourValue)` after the `ActorSystem` is created. I doubt many users will need to do that and changing this default likely works to the benefit of most.